.github/workflows/rust.yml

```diff
@@ -37,7 +37,7 @@ permissions:
 jobs:
   test:
     runs-on: ubuntu-latest-16-cores
-    timeout-minutes: 90
+    timeout-minutes: 95
```

Comment:
Is that still needed, I thought that we decreased the need for such long times.

Reply:
I don't know where the `90` came from, but the +5 accounts for the runtime of `wasm-pack test` on my laptop.